core/types/transaction.go

```diff
@@ -50,8 +50,7 @@ type txdata struct {
 	Price        *big.Int        `json:"gasPrice" gencodec:"required"`
 	GasLimit     uint64          `json:"gas"      gencodec:"required"`
 	ShardID      uint32          `json:"shardID"  gencodec:"required"`
-	ToShardID    uint32          `json:"toShardID" rlp:"nil"` // for cross-shard tx's destination shard ID; nil means intra-shard tx
```

Comment:
why remove ToShardID? we have discussed in discord.

Reply:
this is PR for john. if i remove that and use the new version of the sdk, it's not working. i created this for John.